pkg/mimirpb/compat.go

```diff
@@ -195,6 +238,106 @@ func fromSpansProtoToSpans(s []BucketSpan) []histogram.Span {
 	return spans
 }
 
+func FromHistogramToHistogramProto(timestamp int64, h *histogram.Histogram) Histogram {
+	return Histogram{
+		Count:          &Histogram_CountInt{CountInt: h.Count},
+		Sum:            h.Sum,
+		Schema:         h.Schema,
+		ZeroThreshold:  h.ZeroThreshold,
+		ZeroCount:      &Histogram_ZeroCountInt{ZeroCountInt: h.ZeroCount},
+		NegativeSpans:  fromSpansToSpansProto(h.NegativeSpans),
+		NegativeDeltas: h.NegativeBuckets,
```

Comment:
We should add comment about not needed `NegativeCounts` here. Same for `PositoveDeltas` and counts.

Reply:
it's explained already in the mimir.proto file in the Histogram

Reply:
added comment